sdk/core/core-amqp/src/errors.ts

```diff
@@ -560,6 +553,19 @@ export function translate(err: AmqpError | Error): MessagingError {
 
   let error: MessagingError = err as MessagingError;
 
+  // OperationTimeoutError is retryable as it indicates potential problem with unexpected delays
+  // and not an issue with the Messaging process
+  if (
+    err instanceof OperationTimeoutError ||
+    // instanceof checks on custom Errors doesn't work without manually setting the prototype within the error.
+    // Must do a name check until OperationTimeoutError is updated, and that doesn't break compatibility
+    // https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work
```

Comment:
I love that this is so well documented!

Reply:
That's thanks to @chradek's handiwork from a few lines below :)

Reply:
hey! credit goes where credit is due :p